src/style.css

```diff
@@ -0,0 +1,157 @@
+/* #region card */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
+
+.card {
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+
+  background: var(--white);
+  border: 1px solid var(--grey);
+  border-radius: 5px;
+
+  --white: #fff;
+  --grey: #f3f3f3;
+  --dark-grey: #616070;
+  --light-blue: #00acdc;
+  --dark-blue: #060b35;
+}
+
+.card__image {
+  align-self: center;
+  margin-top: 16px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  align-self: center;
+  margin-top: 40px;
+}
+
+.card__title-content {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--dark-blue);
+}
+
+.card__code {
+  align-self: flex-start;
+  margin-top: 4px;
+}
+
+.card__code-content {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--dark-grey);
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+  margin-top: 16px;
+}
+
+.card__reviews-number {
+  align-self: center;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--dark-blue);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 24px;
+}
+
+.card__price-content {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--dark-grey);
+}
+
+.card__price-amount {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--dark-blue);
+}
+
+.card__buy-button {
+  display: flex;
+  margin-top: 16px;
+}
+
+.card__buy-link {
+  padding: 11px 69px;
+
+  background: var(--light-blue);
+  border: 1px solid var(--light-blue);
+  border-radius: 5px;
+
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: var(--white);
+
+  text-align: center;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.card__buy-link:hover {
+  background: var(--white);
+  color: var(--light-blue);
+}
+/* #endregion */
+
+/* #region stars */
+.stars {
```

Comment:
move Each BEM block to the separate file

Reply:
Fix it 
![image](https://user-images.githubusercontent.com/97547824/207940797-7e17b8c1-e5f7-4ed0-a1ec-376428186988.png)